samples/EverythingServer/Tools/AnnotatedMessageTool.cs

```diff
@@ -15,25 +15,22 @@ public enum MessageType
     }
 
     [McpServerTool(Name = "annotatedMessage"), Description("Generates an annotated message")]
-    public static IEnumerable<Content> AnnotatedMessage(MessageType messageType, bool includeImage = true)
+    public static IEnumerable<ContentBlock> AnnotatedMessage(MessageType messageType, bool includeImage = true)
     {
-        List<Content> contents = messageType switch
+        List<ContentBlock> contents = messageType switch
         {
-            MessageType.Error => [new()
+            MessageType.Error => [new TextContentBlock()
```

Comment:
Nit: shouldn't be addressed here but I notice we're using redundant parens in our object initializer expressions across this repo. Is this deliberate, or just a holdover from mcpdotnet style?
```suggestion
            MessageType.Error => [new TextContentBlock
```